package.json

```diff
@@ -139,9 +139,9 @@
     "aws4": "1.11.0",
     "axios": "1.6.2",
     "bottleneck": "2.19.5",
-    "bull": "4.10.4",
+    "bull": "4.12.8",
```

Comment:
I wonder how tightly coupled this is to @nestjs/bull, looks like they only update their bull dependency since 4.10.4 without needing to update nestjs/bull code. So this should be safe.

We will then do nestjs updates of nestjs/bull and nestjs/cache-manager during the annual monorepo upgrade.